src/source.jl

```diff
@@ -65,7 +65,7 @@ from disk until a particular field of the dataframe is accessed.
 
 To copy the entire file into memory, instead use `materialize`.
```

Comment:
This sentence doesn't sound accurate anymore since copies are made when `!use_mmap`.